standard/recommendations/coverage-subset/PER_subset-slice-dimensions.adoc

```diff
@@ -0,0 +1,14 @@
+[[per_slice_sparse_dimension]]
+[width="90%",cols="2,6a"]
+|===
+^|*Permission {counter:per-id}* |*/per/coverage-subset/slice-sparse-dimension*
+^|A |The empty portions of a slice operation on an axis (e.g. time), combined with a trimming operations on other axes (e.g. latitude and longitude)
```

Comment:
maybe extend to: "The empty portions in a result coverage of a slice operation..."